platform/core/src/studies/sortStudy.js

```diff
@@ -1,17 +1,96 @@
+import getSeriesInfo from './getSeriesInfo';
```

Comment:
Can you walk me through the changes in this file? It looks like we added a fair amount of code here, and I want to understand the benefits + usage

Reply:
actually there is nothing new.. I preserved old implementation of sortStudy and exposed what has being used by retrieveMetadaAsync file.. but its always a pleasure to walk you through :D